pkg/model/dev.go

```diff
@@ -254,6 +255,9 @@ type Annotations map[string]string
 // Environment is a list of environment variables (key, value pairs).
 type Environment []EnvVar
```

Comment:
is this type still being used?

Reply:
Yes, we need to keep it to support retrocompatibility for a time